packages/resolver/lib/sources/truffle/index.ts

```diff
@@ -85,21 +82,38 @@ module.exports = class TestSource {
       "AssertIntArray",
       "AssertString",
       "AssertUint",
-      "AssertUintArray"
+      "AssertUintArray",
+      "NewSafeSend",
+      "OldSafeSend"
     ];
 
-    for (const lib of assertLibraries) {
+    for (const lib of truffleLibraries) {
       if (importPath === `truffle/${lib}.sol`) {
-        const body = fse.readFileSync(
-          path.resolve(path.join(__dirname, `${lib}.sol`)),
-          { encoding: "utf8" }
-        );
+        const actualImportPath =
+          // @ts-ignore
+          typeof BUNDLE_VERSION !== "undefined"
+            ? path.resolve(path.join(__dirname, `${lib}.sol`))
+            : path.resolve(
+                __dirname,
+                "../../../..",
+                "lib/truffleLibraries",
+                `${lib}.sol`
+              );
+        const body = fse.readFileSync(actualImportPath, { encoding: "utf8" });
         return { body, filePath: importPath };
       }
     }
+
+    return { body: null, filePath: null };
+  }
+
+  require(importPath: string): null {
+    return null;
   }
 
-  resolveDependencyPath(importPath, dependencyPath) {
-    return dependencyPath;
+  // Here we're resolving from local files to local files, all absolute.
```

Comment:
Does it need to be absolute? Seems more correct not to be.

Reply:
Seems like tests pass without this coming back as absolute! Marking as resolved.